engine/src/main/java/org/terasology/engine/config/flexible/constraints/StringConstraint.java

```diff
@@ -1,11 +1,10 @@
-// Copyright 2021 The Terasology Foundation
+// Copyright 2022 The Terasology Foundation
 // SPDX-License-Identifier: Apache-2.0
 
 package org.terasology.engine.config.flexible.constraints;
 
 import com.google.common.base.Predicates;
 import com.google.common.base.Strings;
-import org.jetbrains.annotations.NotNull;
```

Comment:
This was unexpected. I assume some dependency must have previously pulled in `org.jetbrains.annotations` as transitive, and upgrading made it no longer visible to us?